src/pages/installing-spark/html-installation.mdx

```diff
@@ -0,0 +1,148 @@
+
+1. First, use `npm` to install the Spark package. This will download the
+JavaScript and CSS needed to add the Spark components.
+
+``` shell
+npm install --save-dev @sparkdesignsystem/spark
+```
+
+2. This will download all the JavaScript and CSS needed to render the Spark
+ components into your `node_modules` folder. To access
+ them in your compiled website, import them in your script. Webpack will
+ include them in your bundled site JavaScript.
+
+``` javascript
+// import the Spark JavaScript
+import spark from "@sparkdesignsystem/spark/es5/spark";
+import sparkPrerender from "@sparkdesignsystem/spark/es5/sparkPrerender";
+
+// optional (see below)
+import "@sparkdesignsystem/spark/es5/sparkPolyfills";
+
+// import the Spark CSS
+import "@sparkdesignsystem/spark/_spark.scss";
+
+// initialize Spark
+sparkPrerender();
+spark();
+```
+
+> **sparkPolyfills** - This file loads a few polyfills that Spark requires. If your application is already polyfilling the items below, you don't need to load this file.
+>
+>- `Promise`
+>- `Array.from`
+>- `String.includes`
+>- `Array.find`
+>- `NodeList.forEach`
+>- `classList` on SVG Elements
+
+The Spark assets will now be included when you compile your site. You can
+verify that this is the case by checking the compiled `bundle.css` and
+`main.js` in your solution. The compiled Spark CSS and JavaScript will show
+ here.
+
+``` shell
+npm run build
+```
+
+<div class="sprk-u-Measure">
+  <img
+    class="sprk-u-mbm "
+    style="margin: auto; display: block;"
+    src={bundledSpark}
+    alt="Minified CSS in bundle.css and minified JS in main.js"/>
+</div>
+
+## Configure Spark
+
+3. Finally, in `index.html`, find the `body` element and add the 
+`data-sprk-main` attribute. This attribute is required for some
+Spark components to correctly handle focus changes.
+
+``` html
+<body data-sprk-main>
+  ...
+</body>
+```
+
+> This attribute is required for some Spark components to correctly handle
+focus changes.
+
+## You did it!
+
+You’re ready to start [adding components](./html-add-components) to
+ your site.
+
+## Additional Topics
+
+* The [Icon Installation Guide](./installing-icons) for importing the Spark SVG icon set
+* The [Font Installation Guide](./installing-fonts) for instructions on using the Rocket Sans font
+
+## Troubleshooting
+
+Here are some issues that you might encounter while setting up Spark and
+ how to fix them:
+
+**Issue: `Uncaught ReferenceError: SPRK_CURRENT_VERSION is not defined`**
+
+Make sure you're importing Spark from the correct directory:
+
+``` javascript
+import spark from "@sparkdesignsystem/spark/es5/spark";
+```
+
+This is not the correct directory:
+
+``` javascript
+import spark from "@sparkdesignsystem/spark/spark";
+```
+
+---
+
+**Issue: `WARN Hit error self signed certificate in certificate chain`**
+
+This is often caused by corporate firewall configurations and
+can sometimes be fixed by prefixing `npm install` with 
+`NODE_TLS_REJECT_UNAUTHORIZED=0`.
```

Comment:
I thought we had decided to not include this in here. But, I forget, could you remind me?

Reply:
I vaguely remember discussing it but I don't remember deciding anything

Reply:
I'm fine deleting it if you think that's best

Reply:
can you resync up with @yodasw16  on it?